server/src/main/java/io/spine/server/commandbus/CommandAckMonitor.java

```diff
@@ -112,7 +112,11 @@ private static EventMessage systemEventFor(Status status, CommandId commandId) {
                                      .build();
             case REJECTION:
             default:
-                throw newIllegalArgumentException("Invalid status %s.", status.getStatusCase());
+                throw newIllegalArgumentException(
```

Comment:
Has this error popped up in your log? If so, this is a bug or a compatibility issue worth paying attention to. Let's discuss if needed.

Reply:
Nope, not really. I looked over the other messages that could have a bit more useful info.